commands/db.bee.inc

```diff
@@ -110,11 +119,71 @@ function db_import_bee_callback($arguments, $options) {
   }
 }
 
+/**
+ * Command callback: Drop and re-create the existing configured database.
+ */
+function db_drop_bee_callback($arguments, $options) {
+  global $_bee_yes_mode;
+  // Initialize our pipes variable.
+  $pipes = array();
+
+  // Get database info.
+  $db_connection = Database::getConnectionInfo();
+  $db_info = $db_connection['default'];
+
+  if (!$_bee_yes_mode) {
+    // Prompt to continue.
+    if (!bee_confirm(bt('Are you sure you want to drop the contents of !database ?', array(
+        '!database' => $db_info['database']
+      )), FALSE)) {
+      return;
+    }
+  }
+
+  $options = '--user=' . rawurldecode($db_info['username']);
+  $options .= " --password='" .  rawurldecode($db_info['password']) . "'";
+  $options .= ' --host=' . $db_info['host'];
+  $db_database = rawurldecode($db_info['database']);
```

Comment:
Following changes from #261 and #282 we can now use the helper function.  Also, we have the database name variable above.

```suggestion
  // Get database connection details.
  $connection_details = db_bee_mysql_options($db_info, FALSE);
  $connection_string = $connection_details['connection'];
  $connection_file = $connection_details['filename'];
```